websites/M/Maliki/dist/metadata.json

```
@@ -0,0 +1,40 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.7",
+	"author": {
+		"name": "RisingSunLight",
+		"id": "240521747852558347"
+	},
+	"service": "Maliki",
+	"description": {
+		"en": "Presence which helps you to follow your activity on the official website of Maliki !",
+		"fr": "Présence qui permet de suivre votre activité sur le site officiel de Maliki !"
```

these are descriptions of the website, not the presence...
```suggestion
		"en": "I hate rambling about myself... Apart from in my comics, my novels and on my blog that is!",
		"fr": "J'ai horreur de parler de moi... Ailleurs que dans mes BD, mes romans et sur mon blog !",
		"es": "Odio hablar de mi... Aparte de mis cómics, mis novelas y en mi blog!"
```

---

Oups, okay, I've didn't understand this in that way!